PROJECT-CLEARWATER-MIB

```diff
@@ -3322,6 +3322,123 @@ sproutSCSCFINVITEsForkedCount OBJECT-TYPE
     DESCRIPTION "The number of extra INVITEs created due to multiple registrations."
     ::= { sproutSCSCFINVITEsForkedEntry 2 }
 
+sproutSCSCFINVITEsNoMatchingDefaultIFCsTable OBJECT-TYPE
+    SYNTAX SEQUENCE OF SproutSCSCFINVITEsNoMatchingDefaultIFCsEntry
+    MAX-ACCESS not-accessible
+    STATUS current
+    DESCRIPTION "Number of initial requests that attempted to use default IFCs but had no matching ones."
+    ::= { sprout 39 }
+
+sproutSCSCFINVITEsNoMatchingDefaultIFCsEntry OBJECT-TYPE
+    SYNTAX      SproutSCSCFINVITEsNoMatchingDefaultIFCsEntry
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION "Number of initial requests that attempted to use default IFCs but had no matching ones."
+    INDEX       { sproutSCSCFINVITEsNoMatchingDefaultIFCsScope }
+    ::= { sproutSCSCFINVITEsNoMatchingDefaultIFCsTable 1 }
+
+SproutSCSCFINVITEsNoMatchingDefaultIFCsEntry ::= SEQUENCE
+{
+  sproutSCSCFINVITEsNoMatchingDefaultIFCsScope        INTEGER,
+  sproutSCSCFINVITEsNoMatchingDefaultIFCsCount        Unsigned32
+}
+
+sproutSCSCFINVITEsNoMatchingDefaultIFCsScope OBJECT-TYPE
+    SYNTAX      INTEGER {
+                  scopePrevious5SecondPeriod(1),
+                  scopeCurrent5MinutePeriod(2),
+                  scopePrevious5MinutePeriod(3)
+                }
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION "The period over which the statistics were collected."
+    ::= { sproutSCSCFINVITEsNoMatchingDefaultIFCsEntry 1 }
+
+sproutSCSCFINVITEsNoMatchingDefaultIFCsCount OBJECT-TYPE
+    SYNTAX      Unsigned32
+    MAX-ACCESS  read-only
+    STATUS      current
+    DESCRIPTION "Number of initial requests that attempted to use default IFCs but had no matching ones."
+    ::= { sproutSCSCFINVITEsNoMatchingDefaultIFCsEntry 2 }
+
+sproutSCSCFINVITEsNoSharedIFCsSetTable OBJECT-TYPE
+    SYNTAX SEQUENCE OF SproutSCSCFINVITEsNoSharedIFCsSetEntry
+    MAX-ACCESS not-accessible
+    STATUS current
+    DESCRIPTION "Number of shared IFCs set IDS retrieved from the HSS where the S-CSCF has no matching set of IFCs."
```

Comment:
The plural IFCs feels bit weird here.
"Number of shared IFC set IDs" might sound better - but I don't mind if you leave this.
I would change "IDS" to "IDs" though.